content/applications/marketing/email_marketing/analyze_metrics.rst

```diff
@@ -17,84 +20,154 @@ locations.
 
-Replied rate
-------------
+  A high received rate can indicate the mailing list used is up-to-date, and the sender
+  authentication is trusted by email providers.
+
+  A low received rate may indicate issues, either with the mailing list used for the mailing, or
+  with the sender authentication. View the :ref:`email-marketing/deliverability-issues` section for
+  more information.
+- :guilabel:`Bounced` rate: This rate measures the percentage of emails that were
+  unsuccessfully delivered, against the total number of sent emails.
+
+  A high bounce rate could indicate issues, either with the mailing list used for the mailing,
+  or with the sender authentication.
+
+  A low bounce rate may indicate that the mailing list used is up-to-date, and the sender
+  authentication is trusted by email providers. View the
+  :ref:`email-marketing/deliverability-issues` section for more information.
+
+.. tip::
+   Click on the respective smart buttons to see the recipient records for each metric.
 
-The *Replied* rate measures the percentage of recipients who responded to the email directly against
-the total number of sent emails.
+.. _email_marketing/create_reports:
 
-A high replied rate may indicate the email resonated with recipients, prompting them to take action
-or provide feedback.
+Create metrics reports
+======================
 
-A low replied rate may suggest that the message lacked relevance or did not contain a clear call to
-action.
+Individual metrics can be analyzed by creating a report. To begin, click on the smart button of the
+desired metric.
 
-Clicked rate
-------------
+Next, click the :icon:`fa-caret-down` :guilabel:`(down arrow)` to the right of the search bar to see
+a drop-down menu of filtering and grouping parameters.
 
-The *Clicked* through rate (CTR), measures the percentage of recipients who clicked on a link within
-the email against the total number of sent emails.
+:guilabel:`Filters`, located in the left column of the search options, can be used to keep only the
+results that fit the filter. For example, selecting the *Bounced* filter only shows emails
+that could not be delivered.
 
-A high |CTR| may indicate the email content was relevant and appropriately targeted. Recipients were
-motivated to click the links provided, and likely found the content engaging.
+:guilabel:`Group By`, found in the middle column, is used to organize the results into groups, and
+can be used with or without filters.
 
-A low |CTR| may indicate issues with either the targeting, or the content itself. Recipients may
-have been unmotivated by the calls to action, if there were any, or the message itself may have been
-directed toward the wrong audience.
+.. note::
+  Setting multiple :guilabel:`Group By` options creates nested groups, according to which option
+  is selected first. For example, selecting :guilabel:`Mailing Lists`, followed by
+  *Bounce*, in the :guilabel:`Group By` column, sorts all results *first* by the mailing
```

Comment:
*Bounce* does not appear to be a readily available Group By option, nor is it listed in the **Add Custom Group** menu. Is this actually a selectable Group By?
![image](https://github.com/user-attachments/assets/47715286-7968-412a-9730-c5ddc8e1446b)

Reply:
A more relevant Group By nesting example here, I'd argue, would be something like **Sent Period** > **Winner Selection** to see Email Marketing campaign "winners" over a specific time frame. 

**Sent Period** > **Responsible** would be another good one to see who on the team is sending in volume or quality (team performance check)